src/config_file.c

```diff
@@ -130,6 +131,24 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 		c->system_bb_statename = g_key_file_get_string(key_file, "system", "barebox-statename", NULL);
 	}
 
+	c->max_bundle_download_size = g_key_file_get_uint64(key_file, "system", "max-bundle-download-size", &ierror);
+	if (g_error_matches(ierror, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND)) {
+		g_message("No value for key \"max-bundle-download-size\" in [system] defined "
+				"- using default value of %d bytes.", DEFAULT_MAX_DOWNLOAD_BUNDLE_SIZE);
+		c->max_bundle_download_size = DEFAULT_MAX_DOWNLOAD_BUNDLE_SIZE;
+	}
+	if (g_error_matches(ierror, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_INVALID_VALUE)) {
+		g_message("Value of key \"max-bundle-download-size\" in [system] "
```

Comment:
We are always very strict in config file parsing. Thus if there is an error in config file, RAUC should also report an error to avoid unintentionally writing configuration that has no effect.

Reply:
Agreed.